app/assets/javascripts/constants/route_constants.es6.js

```diff
@@ -19,6 +19,12 @@ class RoutesSingleton {
       },
     };
   }
+
+  get blueprint() { 
+    return { 
+      home: 'http://www.calblueprint.org/',
+    }
+  };
```

Comment:
Can you put this `get blueprint()` block at the top since it starts with a `b` :+1: